src/app/access-requests/features/access-request-field-edit/access-request-field-edit.component.scss

```diff
@@ -0,0 +1,89 @@
+@use '@angular/material' as mat;
+
+:host {
```

Comment:
I would really keep the labelling and the field separate. The labels need to operate along with other labels, and it's tricky if you put them inside this component. For the field edit component I would simply have the value, the editable input, and the edit/save/cancel buttons. Nothing more. That way we can handle the way we display it/structure the elements as we wish.